CIP-1694/README.md

```diff
@@ -0,0 +1,1161 @@
+
+* `Abstain`
+
+  If an Ada holder delegates to `Abstain`, then their stake is actively marked
+  as not participating in governance.
+
+  The effect of delegating to `Abstain` on chain is that the delegated stake *will not* be considered to be
+  a part of the active voting stake.  However, the stake *will* be considered to be registered for the
+  purpose of the incentives that are described in [Incentives for Ada holders to delegate voting stake](#incentives-for-ada-holders-to-delegate-voting-stake).
+
+* `No Confidence`
+
+  If an Ada holder delegates to `No Confidence`, then their stake is counted as
+  a **no** vote on every governance action apart from a "Motion of no confidence".
+  This also signals that they have no confidence in the existing constitutional committee.
+
+  The effect of delegating to `No Confidence` on chain is that this stake *will* be considered to be
+  a part of the active voting stake. It will count as a `Yes` vote on every `No Confidence`
+  action and a `No` vote on every other action.
+  It also serves as a directly auditable measure of the confidence of Ada holders in the constitutional
+  committee.
+
+
+> **Note**
+> Any Ada holder may register their stake credential as a DRep and delegate to themselves if they wish to actively participate in
+> voting.
+
+#### Registered DReps
+
+In Voltaire, existing stake credentials will be
+able to delegate their stake to registered DReps for voting purposes,
+in addition to the current delegation to stake pools for block production.
+DRep delegation will mimic the existing stake delegation mechanisms (via on-chain certificates).
+Similarly, DRep registration will mimic the existing stake registration mechanisms.
+Additionally, registered DReps will need to vote regularly to still be considered active.
+Inactive DReps do not count towards the active voting stake anymore.
+
+Registered DReps are identified by a credential that can be either:
+
+* A verification key (Ed25519)
+* A native or Plutus script
+
+The blake2b-224 hash digest of a serialized DRep credential is called the _DRep ID_.
+
+The following new types of certificates will be added for governance:
+DRep registration certificates, DRep retirement certificates, and
+vote delegation certificates.
+
+##### DRep registration certificates
+
+DRep registration certificates include:
+
+* a DRep ID
+* an anchor
+
+An **anchor** is a pair of:
+
+* a URL to a JSON payload of metadata
+* a hash of the contents of the metadata URL
+
+The structure and format of this metadata is deliberately left open in this CIP.
+The on-chain rules will not check either the URL or the hash.
+Client applications should, however, perform the usual sanity checks when fetching content from the provided URL.
+
+
+##### DRep retirement certificates
+
+DRep retirement certificates include:
+
+* a DRep ID
+* the epoch number after which the DRep will retire
+
+
+##### Vote delegation certificates
```

Comment:
I'm sure this has been mentioned in one of the comments, but recently I have been thinking about the DRep system. Overall I think it makes sense in a liquid democracy model to be able to delegate your voting rights to an expert of your choosing, especially if you believe that you do not have enough time to review most or all governance actions thoroughly and instead delegate that responsibility. The argument I sometimes saw of "but it takes almost the same amount of time to review what your DReps are doing, you could just vote yourself" is valid. It will also take some time and effort to review your DRep (hopefully, this becomes easier with the right tools), but it's still less time-consuming than reading through each GA. What seems most realistic to me is that most ada holders will just set their delegation once and leave it as that. They won't monitor their DRep as long as no controversial decision is being voted on that affects them, and even then, most won't take any action. This is scary. That means that DReps will do all they can to amass delegations from the Cardano community to boost their decision-making power, think about SPO shilling, but on a whole different level. As no specific parameters are set for the maximum amount of ada a DRep can control, we could see only a handful of DReps ending up on top that decide that fate of us all. A system must be designed to avoid centralization of power, or else, even with the best intentions, it will eventually happen. Look at bitcoin mining and the centralization of hash rate power or the US's current political two-party system. 

So what can we do?
The most straightforward answer to just set a max ada amount per DRep sounds good initially, but this would also mean the idea of the pre-defined DReps has to be tossed, as the rules also apply to them. The question is also, what is the right amount of max ada for this parameter? This way still wouldn't block existing DReps from registering a new DRep certificate, like nobody stops the stake pool operator from opening a new pool. Not my preferred choice.

I'd instead like to see a parameter protocol defining the number of epochs that the DRep delegation certificate is valid. This would require the delegator to renew their vote delegation certificate actively. I was thinking about something similar to KES as the block-producing nodes use, or something similar that would allow the delegators to renew their certificate. That means if a delegator doesn't renew their vote delegation certificate, their stake is not counted as an active voting stake. 

Introducing this parameter of a maximum duration for a valid vote delegation certificate would mitigate the risk of "set and forget" voting delegations. It would leave some responsibility of the governance process with the voters, which should be one of the core duties each Cardano citizen must comply with if they want to participate in governance. As currently proposed, a one-time delegation to a DReps doesn't cut it.

Reply:
yea, putting an ADA cap on the DReps is ineffective without identity verification.

It sounds like you are hoping that if people are forced to re-delegate every so often, that they will pay enough attention and care enough about avoiding centralization that they will delegate away from their current DRep if they see that they have too much stake. Note that this does punish those who want to delegate and forget about the choice for longer than the expiration window.

Reply:
> yea, putting an ADA cap on the DReps is ineffective without identity verification.

This is the same trade-off that was made with stake pools, why not apply the same logic here? I like the idea because it at least creates a big distinction between people who are single DRep operators and those who are multi DRep operators.

I do however like the idea of expiring delegation if the expiration window was set to something like 2-4 years. I don't think that would be a big punishment for those who don't want to check in that often and would hopefully help increase decentralization, and it would probably also help re-delegate ADA away from dead DReps.

Reply:
> This is the same trade-off that was made with stake pools, why not apply the same logic here? I like the idea because it at least creates a big distinction between people who are single DRep operators and those who are multi DRep operators.

With stake pools we introduced an incentives mechanism that was aimed at reducing such splitting, the idea being that people wouldn't want to split their pledge. How that played out is another matter.

> I do however like the idea of expiring delegation if the expiration window was set to something like 2-4 years. I don't think that would be a big punishment for those who don't want to check in that often and would hopefully help increase decentralization, and it would probably also help re-delegate ADA away from dead DReps.

It's not clear to me why this would increase decentralization.

Reply:
Liquid democracy systems often allow transitive delegation, where your delegate can themselves delegate their vote further. This can partially mitigate the issue, because an end-user can delegate instead to someone who they trust to _pick a good final delegate_ rather than to someone they trust to _vote the way they want_. This creates some virtuous division of labour: you have some people (final delegates) who focus on scrutinizing proposals, and some people (intermediary delegates) who focus on scrutinizing final delegates. More generally, you can always delegate by picking someone you _do_ trust and who is _more_ informed than you are.

Reply:
@JaredCorduan 
>Note that this does punish those who want to delegate and forget about the choice for longer than the expiration window.

I believe delegate and forget shouldn't be anything we desire in any way, shape, or form. Having all-mighty DReps that control stake that potentially even the ada holders themselves don't have access to anymore sounds like a dystopia. 

I don't see it as a punishment, primarily if this is implemented instead of the current proposal of locking the ada rewards account if you don't delegate to a DRep. If both of those are implemented, I would call it a punishment, as an ada holder that doesn't want to participate in governance, is forced to delegate to a DRep and then also forced to renew this delegation every X epochs. If only the renewal parameter is implemented, I don't see how that punishes the ada holder. I would rather say it is a punishment for the network as the overall active voting stake gets lowered, making it harder to pass governance actions.

>It's not clear to me why this would increase decentralization.

After or shortly before a delegation certificate expires, the ada holder is forced to take action. This will cause the ada holders to review their current DRep's behavior and how they voted/behaved in the previous period. If the DRep didn't act in the ada holder's best interest or controls too much stake, it is the right time to delegate to another DRep or even for the ada holder to become a DRep. The claim it would increase decentralization is what I hope happens, but of course, it is only a wish.